### DISCUSSION TITLE
Facing issue in deploying an XMLA endpoint

### DISCUSSION THREAD

**Discussion by Pedhisettypavithra** [TabularEditor/TabularEditor3#1150] [Data Modeling]
Hi Team,

we are trying to deploy a metric using XMLA endpoint from Tabular editor. While adding deploy we are facing the below error as per the attachment.
<img width="599" alt="image" src="https://github.com/TabularEditor/TabularEditor3/assets/92840379/bc20e4e1-029a-4287-9867-7eb73e986163">

Thanks & Regards,
Pavithra

**Comment by mlonsk** [comment]
Hi @Pedhisettypavithra 
It looks like you have created an Entity partition in your table instad of a M partition. The Entity partition is a special kind of partition that is for example used in Direct Lake Semantic models. 
I would suggest reviewing your partitions to understand what has happened.
